AsyncDisplayKit/ASDisplayNode+Beta.h

```diff
@@ -38,16 +38,16 @@ ASDISPLAYNODE_EXTERN_C_END
  */
 typedef NS_OPTIONS(NSUInteger, ASDisplayNodePerformanceMeasurementOptions) {
   ASDisplayNodePerformanceMeasurementOptionLayoutSpec = 1 << 0,
-  ASDisplayNodePerformanceMeasurementOptionLayoutGeneration = 1 << 1
+  ASDisplayNodePerformanceMeasurementOptionLayoutComputation = 1 << 1
 };
 
-/**
- * Keys to retrieve performance entries from the performance dictionary.
- */
-extern NSString *const ASDisplayNodeLayoutSpecTotalTimeKey;
-extern NSString *const ASDisplayNodeLayoutSpecNumberOfPassesKey;
-extern NSString *const ASDisplayNodeLayoutGenerationTotalTimeKey;
-extern NSString *const ASDisplayNodeLayoutGenerationNumberOfPassesKey;
+struct ASDisplayNodePerformanceMeasurements
+{
+  CFTimeInterval layoutSpecTotalTime;
+  NSInteger layoutSpecNumberOfPasses;
+  CFTimeInterval layoutComputationTotalTime;
+  NSInteger layoutComputationNumberOfPasses;
+};
```

Comment:
Let's make this a typedef, and then we won't have to use the `struct` keyword everywhere.

```
typedef struct
{
   CFTimeInterval layoutSpecTotalTime;
   NSInteger layoutSpecNumberOfPasses;
   CFTimeInterval layoutComputationTotalTime;
   NSInteger layoutComputationNumberOfPasses;
 } ASDisplayNodePerformanceMetrics;
```